tests/integration/update_cluster/nth_sqs_resources/data/aws_s3_bucket_object_nthsqsresources.example.com-addons-node-termination-handler.aws-k8s-1.11_content

```diff
@@ -168,7 +168,7 @@ spec:
         - name: TAINT_NODE
           value: "false"
         - name: JSON_LOGGING
-          value: "false"
+          value: "true"
```

Comment:
Are we ok with changing to JSON logging?

Reply:
I think so. The older DaemonSet variant further down has always had JSON logging enabled. This change just make the Deployment variant consistent.

Reply:
👍